scripts/docs/create-licenses/README.md

```diff
@@ -0,0 +1,5 @@
+# create-licenses
```

Comment:
Does this need license :)

Reply:
I assumed the README file does not need a license - does it? If so, which license should it have?